lib/ezfile/classes/ezgzipzlibcompressionhandler.php

```diff
@@ -51,11 +51,15 @@ function compressionLevel()
     static function isAvailable()
     {
         $extensionName = 'zlib';
+
+        // dl() is removed in PHP 7.0, see https://secure.php.net/manual/en/function.dl.php
```

Comment:
can rather change code below to check if dl() is defined or not, and keep comment before the if stating that dl has been gradually removed since 5.3 and fully removed in 7.0